plan9/mkfile

```diff
@@ -143,8 +425,10 @@ man:V:		$perlpods pod/pod2man.PL perl
 			for (i in $histpodnames) pod/pod2man pod/$i.pod > $installman3dir/$i
 			
 nuke clean:V:	
-		rm -f *.$O   $extensions^.pm config.sh $perllib config.h $perlshr perlmain.c perl miniperl $archlib/Config.pm $ext_c
-		rm -rf $privlib/IO
+        mv util.c util_orig.c
+		rm -f *.$O config.sh $perllib config.h $perlshr perlmain.c perl miniperl git_version.h $ext_c $more_ext_c generate_uudmap uudmap.h bitcount.h mg_data.h plan9.c plan9ish.h math.h bsd_glob.^(c h) poll.^(c h) re_*.^(c h) uni.data multicall.h
+        mv util_orig.c util.c
+		rm -rf privlib archlib
```

Comment:
why doing a backup&restore `util.c`? should not it not be listed in the cleanup files?

Reply:
@atoomic, ideally, yes, but $ext_c includes all the .c files for the extensions: so Util.xs maps to util.c when on a file system that doesn't preserve case (as sometimes happens, and in my case does happen) it'll remove util.c instead of Util.c.